modules/cli/src/main/scala/scala/cli/config/Keys.scala

```diff
@@ -15,7 +15,10 @@ object Keys {
   val sonatypeUser     = new Key.PasswordEntry(Seq("sonatype"), "user")
   val sonatypePassword = new Key.PasswordEntry(Seq("sonatype"), "password")
 
+  val actionableDiagnostics = new Key.BooleanEntry(Seq.empty, "actionable")
```

Comment:
Maybe rename this to `actions`? Basically, users enable or disable suggested actions.